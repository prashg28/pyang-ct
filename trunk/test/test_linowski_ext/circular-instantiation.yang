 module unique {

   namespace "urn:org:example:xml:ct:unique";
   prefix "u";

   import complex-types {prefix "ct"; }

   ct:complex-type  c1 {
        ct:instance c2 {
            ct:instance-type c2;
        }

        leaf l1 {
            type int8;
        }

        leaf l1 {
            type int8;
        }
   }

   ct:complex-type c2 {
       ct:instance c3 {
           ct:instance-type c3;
       }

       leaf l2 {
           type int8;
       }      
   }

   ct:complex-type c3 {
       ct:instance c1 {
           ct:instance-type c1;
       }

       leaf l3 {
           type int8;
       }
   }

   ct:instance c {
       ct:instance-type c1;
   }

   augment "/c" {
       leaf l1 {
           type int8;
       }
   }

   augment "/c" {
       leaf l1 {
           type int8;
       }
   }

}
