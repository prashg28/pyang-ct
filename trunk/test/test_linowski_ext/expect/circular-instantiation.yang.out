circular-instantiation.yang:17: error: DUPLICATE_CHILD_NAME
circular-instantiation.yang:46: error: BAD_NODE_IN_AUGMENT
circular-instantiation.yang:52: error: BAD_NODE_IN_AUGMENT
